build.sbt

```diff
@@ -51,7 +51,7 @@ libraryDependencies ++= Seq(
   "io.circe" %% "circe-core" % circeVersion,
   "io.circe" %% "circe-generic" % circeVersion,
   "org.apache.thrift" % "libthrift" % "0.9.3",
-  "com.twitter" %% "scrooge-core" % "4.18.0",
+  "com.twitter" %% "scrooge-core" % "19.3.0",
```

Comment:
Ouch, that's a big jump. Will approve on the grounds that you'd already know if there were issues!